integration-test-apps/auto-instrumentation/flask/create_flask_app.py

```diff
@@ -7,16 +7,21 @@
 
 import boto3
 import requests
-from flask import Flask, request, session
+from flask import Flask, session
+import logging
 from opentelemetry import trace
 from opentelemetry.sdk.extension.aws.trace.propagation.aws_xray_format import (
     TRACE_ID_DELIMITER,
     TRACE_ID_FIRST_PART_LENGTH,
     TRACE_ID_VERSION,
 )
 
+if os.environ.get('SAMPLE_APP_LOG_LEVEL') == 'ERROR':
+    log = logging.getLogger('werkzeug')
+    log.setLevel(logging.ERROR)
+
 # NOTE: (NathanielRN) Metrics is on hold until 1.50 release
```

Comment:
Nit: Maybe remove the version number in this comment.

Reply:
Removed from both places!